src/main/java/seedu/address/logic/parser/StudyBananasParser.java

```diff
@@ -30,9 +31,13 @@
      * @return the command based on the user input
      * @throws ParseException if the user input does not conform the expected format
      */
-    public Command<? super Model> parseCommand(String userInput) throws ParseException {
+    public Command<? super Model> parseCommand(String userInput, boolean quizIsOngoing) throws ParseException {
```

Comment:
Clarification of the current mechanics:
Once quiz started, user can still go through the quiz switching between quiz and flashcard tabs, but the commands allowed will only be quiz commands?

Reply:
Yup, will implement the tab freezing in these few days